### ISSUE TITLE
cell cluster label with  cell type name

### ISSUE THREAD

**Issue by whiteorchid** [jphe/scTE#6]
Hi,

May I know how to label the cell lineages names for the clusters, does it need an annotation files, like in the Figure 4b of the paper. Is there a gene markers to cell types txt file? 

![image](https://user-images.githubusercontent.com/6839620/95928624-d8941580-0d8f-11eb-9ae3-1b885023a895.png)

Thanks a lot for your kind guidance!

Best,

**Comment by jphe** [comment]
Yes, we annotate the cell types according to the expression of marker genes. You can see the marker gene expression in the Fig. S6 of our manuscript: https://www.biorxiv.org/content/10.1101/2020.07.23.218800v1.supplementary-material

The marker gene list was get from the orignal study: https://www.nature.com/articles/s41586-018-0590-4
